packages/web/nginx.conf

```diff
@@ -1,5 +1,9 @@
 server {
   listen 80;
+  server_name dev.thehub-aubg.com ;
+  return 301 https://dev.thehub-aubg.com$request_uri;
```

Comment:
This line is environment dependant. Please, use $host instead.

[Done right reference](https://github.com/AUBGTheHUB/hub-conf-signup/blob/master/packages/web/nginx.conf)

Reply:
Requesting review of changes